### PR TITLE
Arrays (not ended)

### PR DESCRIPTION
**Pull request by Uzik17**

### CODE REVIEW

File: yberezhnoy/src/com/octonion/lesson2/class_project/Company.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                System.out.println(tp.getPlan());
+                plansTr.add(tp);
+            }
+        }
+
+        return plansTr;
+    }
+
+
+    public void sortTP() {
```
Comment (bettykatya) [inline yberezhnoy/src/com/octonion/lesson2/class_project/Company.java:48]: it would be nice to call this method in main

**Review by bettykatya — CHANGES_REQUESTED** [review]

File: yberezhnoy/src/com/octonion/lesson2/class_project/Mobile_con.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.octonion.lesson2.class_project;
+
+
+public class Mobile_con {
```
[the remark this answers is not in the document]
Comment (Uzik17) [reply yberezhnoy/src/com/octonion/lesson2/class_project/Mobile_con.java:4]: Deleted class

File: yberezhnoy/src/com/octonion/lesson2/class_project/Mobile_con.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    private String plan;
+    //public int minutes;
```
[the remark this answers is not in the document]
Comment (Uzik17) [reply yberezhnoy/src/com/octonion/lesson2/class_project/Mobile_con.java:6]: class Mobile_con is removed

File: yberezhnoy/src/com/octonion/lesson2/class_project/TariffPlan.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    protected Double inet;
+
+    public TariffPlan(String plan, int customers, double price, Double inet) {
+        this.plan = plan;
+        this.customers = customers;
+        this.price = price;
+        this.inet = inet;
+    }
+
+    public TariffPlan() {
```
[the remark this answers is not in the document]
Comment (Uzik17) [reply yberezhnoy/src/com/octonion/lesson2/class_project/TariffPlan.java:17]: removed

File: yberezhnoy/src/com/octonion/lesson2/class_project/Company.java [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.List;
+
+public class Company {
+    private int customers;
+    private double price;
+
+
+    private List<TariffPlan> plans = new ArrayList();
+
+    public List getPlans() {
```
[the remark this answers is not in the document]
Comment (Uzik17) [reply yberezhnoy/src/com/octonion/lesson2/class_project/Company.java:13]: removed

File: yberezhnoy/src/com/octonion/lesson2/class_project/Company.java [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        return plans;
+    }
+
+    public int getCustomers() {
+        return customers;
+    }
+
+    public double getPrice() {
```
[the remark this answers is not in the document]
Comment (Uzik17) [reply yberezhnoy/src/com/octonion/lesson2/class_project/Company.java:21]: removed

File: yberezhnoy/src/com/octonion/lesson2/class_project/Main.java [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        SimpleTP comfort_M = new SimpleTP("comfort_M", 3458, 10.0, 120, 1000, 20.90);
+        SimpleTP comfort_L = new SimpleTP("comfort_L", 2319, 25.0, 240, 2000, 29.90);
+        SimpleTP comfort_XL = new SimpleTP("comfort_XL", 1109, 50.0, 1000, 5000, 39.90);
+
+        UnlimTP unlimited_XS = new UnlimTP("unlimited_XS", 2314, Double.POSITIVE_INFINITY, 1, 8.90);
+        UnlimTP unlimited_S = new UnlimTP("unlimited_S", 1234, Double.POSITIVE_INFINITY, 2, 12.90);
+        UnlimTP unlimited_M = new UnlimTP("unlimited_M", 2242, Double.POSITIVE_INFINITY, 4, 17.90);
+
+        System.out.println(comfort_L.inet);
+        System.out.println("Unlimited = " + unlimited_M.inet); //??? здесь не очень понятно как инфинити сделать. через Double
```
[the remark this answers is not in the document]
Comment (Uzik17) [reply yberezhnoy/src/com/octonion/lesson2/class_project/Main.java:31]: fixed
sout println for inet are removed (the output was for tests)

File: yberezhnoy/src/com/octonion/lesson2/class_project/Company.java [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+
+    public void addTp(TariffPlan tp) {
+        plans.add(tp);
+        customers += tp.getCustomers();
+        price += tp.getPrice();
+    }
+
+    public ArrayList<TariffPlan> getTpPrices(double left, double right) {
```
[the remark this answers is not in the document]
Comment (Uzik17) [reply yberezhnoy/src/com/octonion/lesson2/class_project/Company.java:32]: renamed to getPriceRange

File: yberezhnoy/src/com/octonion/lesson2/class_project/Company.java (hunk 1 above)
Comment (Uzik17) [reply yberezhnoy/src/com/octonion/lesson2/class_project/Company.java:41]: added  velcom.sortTP() in main

**Pull request merged**